cv_next/hooks/useAPIFetch.ts

```diff
@@ -0,0 +1,33 @@
+import { useCallback } from "react";
+
+export const CVS_API_BASE = "/api/cvs";
```

Comment:
These should probably sit in definitions rather than the specific api files; something like API_DEFINITIONS.